mumerge.py

```diff
@@ -852,14 +882,17 @@ def bed_line_formatter(chromosome, mu_sig_list, width=1.0):
             
             # Status counter and update at stdout
             if verbose:
-                sys.stdout.write("\rProcessed {} of {} regions"
+                sys.stdout.write("\nProcessed {} of {} regions"
```

Comment:
This was meant to be a carriage return (i.e. `\r`), so that it continually overwrites the same line in stdout.